packages/jest-get-type/src/index.ts

```diff
@@ -20,9 +20,18 @@ type ValueType =
   | 'symbol'
   | 'undefined';
 
+const PRIMITIVES = new Set<ValueType>([
+  'string',
+  'number',
+  'boolean',
+  'null',
+  'undefined',
+  'symbol',
```

Comment:
I've already merged this, but I'm not sure if I agree `Symbol` is a primitive type?

/cc @mattphillips

Reply:
https://leanpub.com/understandinges6/read#leanpub-auto-symbols-and-symbol-properties

> Symbols are a primitive type introduced in ECMAScript 6, joining the existing primitive types: strings, numbers, booleans, null, and undefined.

Reply:
awesome!

Reply:
thanks for the link 🙏